ui/src/pages/dynamic-instance.js

```diff
@@ -146,10 +155,17 @@ export default class DynamicInstancePage extends React.Component {
                                         <ServiceNameHeader
                                             domain={domain}
                                             service={service}
+                                            serviceHeaderDetails={
+                                                this.props.serviceHeaderDetails
+                                            }
                                         />
+
                                         <ServiceInstanceDetails
-                                            details={
-                                                this.props.serviceHeaderDetails
+                                            instanceDetailsMeta={
+                                                this.props.instanceDetails
+                                            }
+                                            categoryType={
+                                                SERVICE_TYPE_DYNAMIC_LABEL
```

Comment:
we should use `SERVICE_TYPE_DYNAMIC` instead of `SERVICE_TYPE_DYNAMIC_LABEL`

Reply:
done